drivers/ovsdriver.go

```diff
@@ -377,44 +373,64 @@ func (d *OvsDriver) DeleteEndpoint(id string) (err error) {
 	return nil
 }
 
-// CreatePeerHost creates peer host object and adds VTEPs if necessary
-func (d *OvsDriver) CreatePeerHost(id string) error {
-	log.Infof("CreatePeerHost for %s", id)
+// AddPeerHost adds VTEPs if necessary
+func (d *OvsDriver) AddPeerHost(node core.ServiceInfo) error {
+	// Nothing to do if this is our own IP
+	if node.HostAddr == d.localIP {
+		return nil
+	}
+
+	log.Infof("CreatePeerHost for %+v", node)
```

Comment:
Once the team size grows, we can follow this kind of process. In early stages of the project, its best to let the module owners do the right thing. I have full confidence in this team that all of us will do the right thing :)